psst-gui/src/ui/artist.rs

```diff
@@ -258,7 +258,7 @@ impl<T: Data, B: Widget<T>, S: Widget<T>> Widget<T> for ArtistInfoLayout<T, B, S
 
     fn layout(&mut self, ctx: &mut LayoutCtx, bc: &BoxConstraints, data: &T, env: &Env) -> Size {
         let max = bc.max();
-        let wide_layout = max.width > theme::grid(60.0);
+        let wide_layout = max.width > theme::grid(60.0) + theme::GRID * 2.0;
```

Comment:
I tried this and you'll see that it doesn't work I think because the width of the box constraints is different. @SO9010

Reply:
Ah, I didn't realise this as I'm using a tiling manager. I have updated it, but it may be a bit brute force as I just tested values until it matched.